FaCT++.Python/CMakeLists.txt

```diff
@@ -1,22 +1,19 @@
 set(CMAKE_VERBOSE_MAKEFILE ON)
 cmake_minimum_required(VERSION 3.13.0)
 set(CMAKE_CXX_STANDARD 11)
-find_program(PYTHON "python")
+set(PYTHON "python")
 
-if (PYTHON)
```

Comment:
Seems to be fine; however, what if a user does not have Python installed on the system? Previously the build system worked in such a way that `factpp` was able to be built from the root directory even if Python bindings were not needed. However, this PR makes it impossible. If you are OK with that, feel free to merge